src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -3643,6 +3644,58 @@ def server_aad_only_enable(
         azure_ad_only_authentication=True
     )
 
+###############################################
+#           sql server trust groups           #
+###############################################
+
+
+def server_trust_group_create(
+        client,
+        resource_group_name,
+        name,
+        location,
+        group_member,
+        trust_scope):
+
+    members = [ServerInfo(server_id=member) for member in group_member]
+    return client.create_or_update(resource_group_name=resource_group_name,
```

Comment:
Is it a long running operation?

Reply:
I would say yes

Reply:
In this way, we will recommend to use sdk_no_wait() to support no wait for long running operation as below:
![image](https://user-images.githubusercontent.com/49508232/111431358-a586f100-8736-11eb-8ec2-93e9d48b9795.png)

For more information, see https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands (4)